docs/gulp.md

```diff
@@ -77,4 +77,11 @@ The `fix` command runs the syntax fixers provided by the code linters used for
 validating SCSS and JavaScript (ES6). The command will try its _best_ to fix
 coding standard violations based on specific rulesets for each language.
 
+# Gulp Addons
+
+## Autoprefixer
```

Comment:
I think this should live under the `## Build command` section since it's related to building the scss files.

Reply:
I might agree, only because it is part of the build step and if you dont know why vendor prefixes are being added during build, that's the place to look first

Reply:
I updated the wording here, I think it makes sense to keep additional functionality that may or may not be used per project in a separate section that we can highlight. Let me know if this works.